api/client/src/.openapi-generator/FILES

```diff
@@ -164,6 +164,4 @@ setup.cfg
 setup.py
 test-requirements.txt
 test/__init__.py
-test/test_login_nodes_pool.py
-test/test_login_nodes_state.py
```

Comment:
I believe these was wrongly added by https://github.com/aws/aws-parallelcluster/commit/0e241c6786b24ad56e4c4c9fecacc8912c6dafd9.
This file is auto-generated and should not be modified manually